go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -541,7 +541,7 @@ func CreateVReplicationTable() []string {
   id INT AUTO_INCREMENT,
   workflow VARBINARY(1000),
   source VARBINARY(10000) NOT NULL,
-  pos VARBINARY(10000) NOT NULL,
+  pos BLOB NOT NULL,
```

Comment:
In order for in-place upgrades to also get the new column definition, we need to add an `ALTER TABLE ... MODIFY ...`
(lines 561-566).
I see we did that for `source` but not sure why the definition here still says `VARBINARY`.